src/server/editorServices.ts

```diff
@@ -450,7 +450,7 @@ namespace ts.server {
             if (this.host.realpath) {
                 this.realpathToScriptInfos = createMultiMap();
             }
-            this.currentDirectory = this.host.getCurrentDirectory();
+            this.currentDirectory = normalizePath(this.host.getCurrentDirectory());

```

Comment:
Is it feasible to set `currentDirectory: NormalizedPath` now so that the type expresses that it's already been normalized?

Reply:
Sure. Wasn't familiar with `NormalizedPath` (still don't quite get it, as it doesn't seem to offer any interesting typechecking?)

Reply:
I think it's pretty underpowered right now but I'd like it to be more helpful in the future; seems like many of the path-related bugs I've had to fixed would have been caught by more consistent usage.